tillerinobot/src/main/java/tillerino/tillerinobot/lang/Italiano.java

```diff
@@ -170,7 +179,7 @@ public String invalidAccuracy(String acc) {
  
 	@Override
 	public Response optionalCommentOnLanguage(OsuApiUser apiUser) {
-		return new Message("marcostudios e Howl mi hanno insegnato l'italiano <3");
+		return new Message("[https://osu.ppy.sh/u/-_Marco_- - Marco -] e [https://osu.ppy.sh/u/Howl Howl] mi hanno insegnato l'italiano <3");
```

Comment:
how you did it first on your name was correct, what i meant was use the numeric id url, so "https://osy.ppy.sh/123" instead of "https://osy.ppy.sh/abc", sorry for confusion ;__;

howl's userid is 2751672, so "https://osu.ppy.sh/u/2751672"